src/modules/movie-cards.js

```diff
@@ -0,0 +1,85 @@
+/* eslint-disable no-underscore-dangle */
```

Comment:
- [ ] As suggested by the reviewer [here](https://github.com/victormc13/JavaScript-Capstone/pull/26#discussion_r1283890710), kindly make sure that linters are not disabled. I truly understand the amount of hard work you are putting into this, but linters play a crucial role in maintaining code quality and adhering to best practices. By keeping linters enabled, we can ensure that the codebase remains consistent, easier to maintain, and less prone to errors in the long run. ⛏🤓 You are almost done; please make sure to fix. 📌✨

  _I tried running the app without disabling linters, and it's working as expected. 
You can reach out to me on Slack and we can fix this._